acceptance/framework/consul/cli_cluster.go

```diff
@@ -118,29 +127,24 @@ func (c *CLICluster) Create(t *testing.T) {
 	})
 
 	// Set the args for running the install command.
-	args := []string{"install"}
+	args = append([]string{"install"}, mergeArgs(cliDefaultArgs, args...)...)
 	args = c.setKube(args)
-
 	for k, v := range c.values {
 		args = append(args, "-set", fmt.Sprintf("%s=%s", k, v))
 	}
-
-	// Match the timeout for the helm tests.
-	args = append(args, "-timeout", "15m")
-	args = append(args, "-auto-approve")
+	args = append(args, "-namespace", c.namespace)
 
 	out, err := c.runCLI(args)
 	if err != nil {
 		c.logger.Logf(t, "error running command `consul-k8s %s`: %s", strings.Join(args, " "), err.Error())
 		c.logger.Logf(t, "command stdout: %s", string(out))
 	}
-	require.NoError(t, err)
```

Comment:
See my comment below where I realized why you were doing it 😄 

In addition to that, there are other tests for CLI that rely on this error checking. If we remove it, there's no way to make sure that this function succeeded because it does not return an error. This is another sign that acceptance tests framework is not meant for testing failing installations. So in your case, you'd have to either add another method that returns an error to the interface and implementations (e.g. `CreateE(t *testing.T) error`) or not test it in acceptance tests like I mentioned in another comment.